Help on completing this project proposal document can be found in the
“working with us “

section of the BABS website
<https://bioinformatics.thecrick.org/babs/working-with-us/>

<table>
<thead>
<tr class="header">
<th>Budget code</th>
<th>CC2186</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Time estimate</td>
<td>To be agreed on after project discussions</td>
</tr>
<tr class="even">
<td>Analysis goals</td>
<td><p><strong>Please provide an outline of the analysis goals:</strong></p>
<p>The expected outcomes will be: (a) polished figures (in PDF format) for inclusion in a manuscript, alongside (b) any data analysis tables, and (c) data deposited in accessible server system, eg GEO. Ideally, sharing of R code and intermediate data would be hugely helpful as the project progresses, so we can work on things together.</p></td>
</tr>
<tr class="odd">
<td>Analysis details</td>
<td><p><strong>Please provide details of the data analysis required.</strong> </p>
<p><strong>Dataset</strong></p>
<p>Chris Cooke (PhD in Moris lab) has been working on characterising a population of primordial germ cell like cells (PGCLCs) in mouse gastruloids. He has generated some pilot 10x data and has frozen cells from a timecourse of FACS sorted cells that we plan to 10x sequence as soon as possible. I have already done some basic analysis of the pilot dataset, but am awaiting the additional populations to undergo sequencing.</p>
<p>Biologically, we are interested to know what these sorted populations of cells are, whether they are equivalent to embryonic PGCs, and particularly, which timepoint they are most similar to. As such, the analysis revolves around initial basic exploration of the timecourse, comparison to similar (smaller) datasets from the same in vitro model, and then comparison to embryonic datasets from different stages of development, as described below.</p>
<p>Full descriptions of the sorted populations are provided in attached spreadsheet.</p>
<p><strong>Done so far:</strong></p>
<ol type="1">
<li><p>Pilot dataset analysis</p>
<ol type="a">
<li><p>Initial QC check</p></li>
<li><p>Cellranger aggregation of populations together</p></li>
<li><p>Identification of sorted populations (by biological cell type)</p></li>
</ol></li>
</ol>
<p><strong>To do:</strong></p>
<ol type="1">
<li><p>Compare pilot dataset to existing in vitro datasets (see Point 4 below)</p></li>
<li><p>Examine timecourse dataset</p>
<ol type="a">
<li><p>QC of new datasets</p></li>
<li><p>Confirm identity of sorted populations (across timepoints) using <em>a priori</em> marker gene lists [NM to provide]</p></li>
<li><p>How are sorted populations related to one another?</p>
<ol type="i">
<li><p>Any overlap between populations?</p></li>
<li><p>Potential fate decision points or maturation?</p></li>
</ol></li>
<li><p>Does the same sorted population change at all over time?</p></li>
<li><p>Combine timepoint datasets and look for trends in the data</p>
<ol start="3" type="i">
<li><p>Signif DEGs over time</p></li>
<li><p><em>Possible addition:</em> Pseudotime analysis</p></li>
</ol></li>
</ol></li>
<li><p>Compare our data to existing gastruloid datasets [<a href="https://www.nature.com/articles/s41586-020-2024-3">van den Brink, 2020</a>; <a href="https://www.science.org/doi/10.1126/science.aba4937">Veenvliet, 2020</a>]</p>
<ol start="6" type="a">
<li><p>Combine datasets together [MNN?]</p>
<ol start="5" type="i">
<li><p>Do our sorted populations overlap with annotated PGCLCs?</p>
<ul>
<li><p>If not, which populations?</p></li>
</ul></li>
</ol></li>
</ol></li>
<li><p>Compare our data to existing <em>in vivo</em> datasets [<a href="https://www.nature.com/articles/s41556-020-0489-9">Pijuan-Sala, 2020</a>; <a href="https://www.nature.com/articles/s41586-020-2552-x">Grosswendt, 2020</a>; <a href="https://faseb.onlinelibrary.wiley.com/doi/full/10.1096/fj.202002420R">Mayere, 2021</a>; <a href="https://www.nature.com/articles/s41467-021-27172-0">Zhao, 2021</a> --- NB: <em>pull out the annotated PGC populations at early timepoints to reduce datapoints</em>]</p>
<ol start="7" type="a">
<li><p>Do our sorted populations overlap with the PGCs/germ cells?</p></li>
<li><p>How do the timepoints compare to temporal development in vivo?</p></li>
<li><p>What is the most mature stage of PGCs identified in our dataset?</p></li>
<li><p>Is there any sex bias (XX or XY embryos?)</p></li>
<li><p>Is there any difference between our populations and the most similar embryonic population?</p></li>
</ol></li>
<li><p>Compare our data to existing <em>in vitro</em> datasets [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8486853/">Bleckwehl, 2021</a>; <a href="https://www.sciencedirect.com/science/article/pii/S0092867411007719">Hayashi 2011</a> (bulk, microarray)]</p>
<ol start="12" type="a">
<li><p>Do our sorted populations overlap with the PGCLCs?</p></li>
<li><p>How do the timepoints compare to alternative in vitro timepoints?</p></li>
<li><p>Is there any difference between our populations and the most similar in vitro population?</p></li>
</ol></li>
</ol></td>
</tr>
</tbody>
</table>

Please note that the number of hours quoted to carry out the analysis is
an estimate and represents the maximum amount of time you are willing to
commit to the work. We will contact you if this analysis time is reached
and we can decide together whether to commit more time to the work. You
will only be charged for the time taken to carry out the analysis.

We work in line with the [Crick's authorship
policy](https://intranet.crick.ac.uk/our-crick/research-integrity/pages/publication-authorship)
and our expectation is significant contributions to your research are
recognised by authorship. If you have any questions or concerns
regarding authorship please discuss them with us as early as possible.
